starlite/datastructures/multi_dicts.py

```diff
@@ -41,3 +42,41 @@ async def close(self) -> None:
         for _, value in self.multi_items():
             if isinstance(value, UploadFile):
                 await value.close()
+
+
+class QueryMultiDict(MultiDict[Any]):
```

Comment:
The way we parse it, the items can only ever be a `str` or `bool`, so why not annotate it as such?